tests/explicit-nulls/pos/notnull/J.java

```diff
@@ -4,6 +4,7 @@
 public class J {
 
     @Nonnull
+    // TODO: remove annotaion after #7483
```

Comment:
I think we can remove this. Once #7483 is fixed, we can add a new test that checks that final fields sometimes get ConstantTypes. This test is about @Nonnull.